lib/jsonapi-serializers/attributes.rb

```diff
@@ -33,10 +33,14 @@ def attributes(*names)
       end
 
       def has_one(name, options = {}, &block)
+        options[:links] ||= true
```

Comment:
For consistency, move these defaults inside of the `add_to_one_association` or `add_to_many_association` methods instead of out here.